TypeCobol/AntlrGrammar/Cobol2002CodeElements.g4

```diff
@@ -55,3 +58,20 @@ allocateStatement:
 // New Cobol v6.1 FREE statement that releases dynamic storage that was previously obtained with an ALLOCATE statement.
 freeStatement:
 	FREE pointerStorageArea+;
+
+// New Cobol v6.1 JSON GENERATE statement. Allows generating JSON string from data item, similar to XML GENERATE.
+jsonGenerateStatement:
+	JSON GENERATE destination=storageArea1
+	FROM source=variable1
+	(COUNT IN? charactersCount=storageArea1)?
+	(name OF? jsonNameMapping+)? // Re-use of contextual keyword NAME defined for XML GENERATE in CobolCodeElements.
```

Comment:
In our grammar, NAME is a contextual keyword. It was initially created for the XML GENERATE and I simply re-used it for JSON GENERATE because usage is very similar.